### DISCUSSION TITLE
Stand Up Tuesday April 25th

### DISCUSSION THREAD

**Discussion by CristianCazares** [Bugs-io/TC3001B#69] [Main Stand Ups]
```
**What did you achieve in the last 24 hours?**:
 - 
**What are your priorities for the next 24 hours?**:
 - 
**Blockers**:
 -
**Shoutouts**:
 -@username for x
```

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Completely discard NextJS 13
 - Started setting up the Api Gateway

**What are your priorities for the next 24 hours?**:
- Get the back set up on my machine
- Set the endpoints for authentication
- Get started on protecting routes, context, etc.

**Blockers**:
- Back not running on my machine

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Sent PR with login/signup and dotenv loading
 - Worked with Jacob to create the data analysis repository
 
**What are your priorities for the next 24 hours?**:
 - Work with @Jacobgtz024 on the data analysis service
 
**Blockers**:
 - None

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Internal dataset management is done
 
**What are your priorities for the next 24 hours?**:
 - Finished the upload file proccess
 - 
**Blockers**:
 - Not sure about the way I am implementing that
 
**Shoutouts**:
 - None

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
 - Developed a matching algorithm to compare Udemy courses and IBM certs 
 - Started the dummy script that receives a xlsx file and returns a csv file 
**What are your priorities for the next 24 hours?**:
 - Finish dummy script 
 - Turn it into a functional cleaning script
**Blockers**:
 - Potpourri
**Shoutouts**:
 -@todos
